ops/model.py

```diff
@@ -487,15 +486,14 @@ def _invalidate(self):
     def status(self) -> 'StatusBase':
         """Used to report or read the status of a specific unit.
 
-        The status of any unit other than yourself is always Unknown.
+        The status of any unit in a different charm is always Unknown.
```

Comment:
This isn't quite right, as the charm can be deployed on multiple units. Perhaps:
```suggestion
        The status of any unit other than the current unit is always Unknown.
```